input/fsh/Terminology.fsh

```diff
@@ -1,34 +1,36 @@
-// DEMOGRAPHICS CodeSystems
-
 Alias: $FHIRMS = http://terminology.hl7.org/CodeSystem/v3-MaritalStatus
+Alias: $SCT = http://snomed.info/sct // standard set is published separately for re-use in other IGs
```

Comment:
```suggestion
Alias: $SCT = http://snomed.info/sct
```

It's defined in the [FHIR spec](http://build.fhir.org/terminologies-systems.html), so the comment isn't quite on point - better to leave it out.